Jenkinsfile

```
@@ -415,6 +440,7 @@ pipeline {
         SSH_KEY_ARGS = "-ici_key"
         CLUSH_ARGS = "-o$SSH_KEY_ARGS"
         TEST_RPMS = cachedCommitPragma(pragma: 'RPM-test', def_val: 'true')
+        BUILD_OPTION = rpm_faults_args()
```

Let's make this less vague and more direct:
```suggestion
        RPM_FAULTS_ARGS = rpm_faults_args()
```